custom/abdm/milestone_one/views/abha_verification_views.py

```diff
@@ -23,7 +23,7 @@ def get_auth_methods(request):
         error_msg = "Missing required parameter: health_id"
         return generate_invalid_req_response(error_msg)
     resp = abdm_util.search_by_health_id(health_id)
-    auth_methods = resp.get("auth_methods")
+    auth_methods = resp.get("authMethods")
```

Comment:
Can you share they this was changed?

Seems like from commit it was for `searchByHealthId` but we changed it in a common function?

Reply:
We use `searchByHealthId` endpoint to get the auth methods available to a beneficiary. 
Sample response from ABDM
```
{
  "authMethods": [
    "AADHAAR_OTP"
  ],
  "healthId": "string",
  "healthIdNumber": "string",
  "name": "string",
  "tags": {
    "additionalProp1": "string",
    "additionalProp2": "string",
    "additionalProp3": "string"
  }
}
```

Reply:
Right. My question was why was it changed now? Isn't this view being used in the earlier PR.